sql/ddl_api.sql

```diff
@@ -91,11 +91,29 @@ BEGIN
     WHERE attrelid = main_table AND attname = time_column_name;
 
     -- Timestamp types can use default value, integral should be an error if NULL
-    IF time_type IN ('TIMESTAMP', 'TIMESTAMPTZ', 'DATE') AND chunk_time_interval IS NULL THEN
-        chunk_time_interval_actual := _timescaledb_internal.interval_to_usec('1 month');
-    ELSIF time_type IN ('SMALLINT', 'INTEGER', 'BIGINT') AND chunk_time_interval IS NULL THEN
-        RAISE EXCEPTION 'chunk_time_interval needs to be explicitly set for types SMALLINT, INTEGER, and BIGINT'
-        USING ERRCODE = 'IO102';
+    IF time_type IN ('TIMESTAMP', 'TIMESTAMPTZ', 'DATE') THEN
+        IF chunk_time_interval IS NULL THEN
+            chunk_time_interval_actual := _timescaledb_internal.interval_to_usec('1 month');
```

Comment:
I think we should should change the default partitioning from a month to a day or week.  Better for chunks to be too small than too large.  Include with this PR or just have a standalone with one line change?

Reply:
I don't think that changing a default should be mixed in with this PR